platform/ui/src/components/Select/Select.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import ReactSelect from 'react-select';
+
+import './Select.css';
+
+const Select = ({
+  autoFocus,
+  className,
+  classNamePrefix,
+  isDisabled,
+  isMulti,
+  isSearchable,
+  name,
+  onChange,
+  options,
+  placeholder,
+  noOptionsMessage,
```

Comment:
I didn't understand what it does, can you give me a quick example on how to use that?

Reply:
I have added an example into the docz. also update the code, it was a bit wrong. :) 
thanks for the callout

Reply:
Cool! Just checked, I was trying to pass a string instead of a node.

We'll just have to change its prop type definition. `noOptionsMessage` should expect a `node` instead of a `func`.